src/main/java/org/assertj/core/error/ConditionAndGroupGenericParameterTypeShouldBeTheSame.java

```diff
@@ -24,7 +24,7 @@
 public class ConditionAndGroupGenericParameterTypeShouldBeTheSame extends BasicErrorMessageFactory {
 
   public ConditionAndGroupGenericParameterTypeShouldBeTheSame(Object actual, Condition<?> condition) {
-    super("%nExpecting: <%s> have the same generic type as condition <%s>", actual, condition);
+    super("%nExpecting: %s have the same generic type as condition %s", actual, condition);
```

Comment:
I realize there is a missing `to` here, should be `Expecting: %s to have`

Reply:
Yes, I am adding them.